src/test/java/integ/com/mathworks/ci/RunMatlabCommandBuilderTest.java

```diff
@@ -306,65 +306,63 @@ public void verifyDefaultMatlabNotPicked() throws Exception {
+            jenkins.assertLogContains("run-matlab-command", build);
+            jenkins.assertBuildStatus(Result.FAILURE, build);
+            vals.put("VERSION", "R2015b");
+            Combination c2 = new Combination(vals);
+            MatrixRun build2 = matrixProject.scheduleBuild2(0).get().getRun(c2);
+            jenkins.assertLogContains("MatlabNotFoundError", build2);
+            jenkins.assertBuildStatus(Result.FAILURE, build2);
+    }
 
-	/*
-	 * Test to verify if Matrix build passes (mock MATLAB).
-	 */
-	@Test
-	public void verifyMatrixBuildPasses() throws Exception {
-		MatrixProject matrixProject = jenkins.createProject(MatrixProject.class);
-		Axis axes = new Axis("VERSION", "R2018a", "R2018b");
-		matrixProject.setAxes(new AxisList(axes));
-		String matlabRoot = getMatlabroot("R2018b");
-        this.buildWrapper.setMatlabBuildWrapperContent(new MatlabBuildWrapperContent(Message.getValue("matlab.custom.location"), matlabRoot.replace("R2018b", "$VERSION")));
-		matrixProject.getBuildWrappersList().add(this.buildWrapper);
-		RunMatlabCommandBuilderTester tester = new RunMatlabCommandBuilderTester(matlabExecutorAbsolutePath,
-				"-positive");
-
-		tester.setMatlabCommand("pwd");
-		matrixProject.getBuildersList().add(tester);
-		MatrixBuild build = matrixProject.scheduleBuild2(0).get();
-
-		jenkins.assertLogContains("R2018a completed", build);
-		jenkins.assertLogContains("R2018b completed", build);
-		jenkins.assertBuildStatus(Result.SUCCESS, build);
+    /*
+     * Test to verify if Matrix build passes (mock MATLAB).
+     */
+    @Test
+    public void verifyMatrixBuildPasses() throws Exception {
+            MatrixProject matrixProject = jenkins.createProject(MatrixProject.class);
+            Axis axes = new Axis("VERSION", "R2018a", "R2018b");
+            matrixProject.setAxes(new AxisList(axes));
+            String matlabRoot = getMatlabroot("R2018b");
+            this.buildWrapper.setMatlabBuildWrapperContent(new MatlabBuildWrapperContent(Message.getValue("matlab.custom.location"), matlabRoot.replace("R2018b", "$VERSION")));
+            matrixProject.getBuildWrappersList().add(this.buildWrapper);
+            RunMatlabCommandBuilderTester tester = new RunMatlabCommandBuilderTester(matlabExecutorAbsolutePath,
+                            "-positive");
+
+            tester.setMatlabCommand("pwd");
+            matrixProject.getBuildersList().add(tester);
+            MatrixBuild build = matrixProject.scheduleBuild2(0).get();
+
+            jenkins.assertLogContains("R2018a completed", build);
+            jenkins.assertLogContains("R2018b completed", build);
+            jenkins.assertBuildStatus(Result.SUCCESS, build);
 	}
 	
-	/*
+    /*
      * Test to verify if command parses succesfully when multiple combinations of 
      * characters are passed. (candidate for integ-tests once integrated)
      */
-
-    
     public void verifyMultispecialChar() throws Exception {

```

Comment:
This test doesn't have `@Test` annotation and doesn't run. If I add it the test fails. Will look into it a little futher

Reply:
The tests under src/test/java/integ are failing because of incorrect test set up. I am working on adding system tests for Jenkins plugin. As part of the same task, I will either delete/update these integ tests.

Reply:
Sounds good. Thanks for the update, Vahila!